src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/Parsers/ExpressionParser.cs

```diff
@@ -265,11 +265,10 @@ private Expression ParseChainedExpression(string resourceType, SearchParameterIn
 
         private Expression ParseSearchValueExpression(SearchParameterInfo searchParameter, string modifier, string value)
         {
-            SearchModifierCode? parsedModifier = ParseSearchParamModifier();
-
+            SearchModifier parsedModifier = ParseSearchParamModifier();
             return _searchParameterExpressionParser.Parse(searchParameter, parsedModifier, value);
 
-            SearchModifierCode? ParseSearchParamModifier()
+            SearchModifier ParseSearchParamModifier()
```

Comment:
>ParseSearchParamModifier [](start = 27, length = 24)

nit: Maybe it's worth to move this function which creates `SearchModifier` to `SearchModifier` itself?

Reply:
actually, disregard that comment, it would be much easier to convert that class to record as we moved to .net 5, and it a only place where we do parsing anyway.